qoe-metrics-dashboard/src/app/models/types/metrics/consumption-report.interface.ts

```diff
@@ -0,0 +1,30 @@
+export type TConsumptionOverviewReportResponse = IConsumptionDetailReport[];
+
+export interface IConsumptionDetailReport {
+    mediaPlayerEntry: string;
+    reportingClientId: string;
+    consumptionReportingUnits: ConsumptionReportingUnit[];
+    stability: number;
+}
+
+export interface ConsumptionReportingUnit {
+    mediaConsumed: string;
+    clientEndpointAddress: ClientEndpointAddress;
+    serverEndpointAddress: ServerEndpointAddress;
+    startTime: string;
+    duration: number;
+    locations: any[];
```

Comment:
could that be typed better, or otherwise i wouldnt include this in the interface

Reply:
Good that you looked at it.
locations is in all `.json`files empty (`[]`) => therefor I now changed it to `never[]`. Is this okay?